index.html

```diff
@@ -124,5 +124,32 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-section">
+    <div id="join-hiring">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <form action="#">
+        <button type="submit" id="roles-button">See the roles</button>
+      </form>
```

Comment:
Why are you using a form when no input is being taken? Where's the anchor for the link? 

It doesn't seem like you've actually used the website before trying to make a clone of this part. You'll see why it doesn't need any forms if you actually use it and see what it's doing.